contracts/evmscript/EVMScriptRegistry.sol

```diff
@@ -44,6 +44,12 @@ contract EVMScriptRegistry is IEVMScriptRegistry, EVMScriptRegistryConstants, Ar
         DisableExecutor(_executorId, executorEntry.executor);
     }
 
+    function enableScriptExecutor(uint256 _executorId) external auth(REGISTRY_MANAGER_ROLE) {
```

Comment:
Wondering if we should add a parameter to the `REGISTRY_MANAGER_ROLE` as it can now do 3 different actions with different security implications. Alternatively we can split off the enabling/disabling functionality into a separate role. Thoughts?

Reply:
Agreed on the enabling / disabling aspect having different security assumptions.